modules/profiling/src/main/java/org/apache/ignite/internal/profiling/util/ProfilingDeserializer.java

```diff
@@ -0,0 +1,294 @@
+/*
```

Comment:
Need to add a test that validates serializing-deserializing of the operations. There is no need to start Ignite, just an instance of LogFileProfiling writing all possible operations and then an instance of ProfilingDeserializer reading those logged operations.